CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to Contribute to the Project
```

Comment:
Maybe we should adapt this at some point, but for now I'm fine with adding *as-is* to have something in place asap.